diff --git a/Eval_reach_int.py b/Eval_reach_int.py
index 0d0a300..bbff488 100644
--- a/Eval_reach_int.py
+++ b/Eval_reach_int.py
@@ -116,7 +116,6 @@ for i in tqdm(range(trial)):
           #obs = np.stack([obs, obs, obs])
           #obs = env.get_obs_dict()        
           action, _ = model.predict(obs, deterministic=False)
-          print(action)
           obs, reward, done, info = env.step(action)
           solved = info['solved']
           if i < trial:
diff --git a/Train_simple_rgb.py b/Train_simple_rgb.py
index db044d8..c752d55 100644
--- a/Train_simple_rgb.py
+++ b/Train_simple_rgb.py
@@ -100,12 +100,12 @@ time_now = datetime.now().strftime("%Y_%m_%d_%H_%M_%S")
 IS_WnB_enabled = True
 env_name = 'UR10eReachFixed-v12'
 
-training_steps = 2000000
+training_steps = 5000000
 ENTROPY = 0.01
 LR = linear_schedule(0.0003)
 CR = linear_schedule(0.1)
 
-loaded_model = 'N/A'
+loaded_model = '2024_12_11_10_10_44'
 try:
     import wandb
     from wandb.integration.sb3 import WandbCallback
@@ -154,7 +154,7 @@ print(time_now)
 # Create a model using the vectorized environment
 #model = SAC("MultiInputPolicy", envs, buffer_size=1000, verbose=0)
 model = PPO(CustomMultiInputPolicy, envs, ent_coef=ENTROPY, learning_rate=LR, clip_range=CR, n_steps = 2048, batch_size = 64, verbose=0, tensorboard_log=f"runs/{time_now}")
-#model = PPO.load(r"./Reach_Target_vel/policy_best_model/" + 'UR10eReachFixed-v0' + '/' + loaded_model + '/best_model', envs, verbose=0, tensorboard_log=f"runs/{time_now}")
+#model = PPO.load(r"./Reach_Target_vel/policy_best_model/" + env_name + '/' + loaded_model + '/best_model', envs, verbose=0, tensorboard_log=f"runs/{time_now}")
 
 callback = CallbackList([eval_callback])
 
diff --git a/Train_simple_rgb2.py b/Train_simple_rgb2.py
index b2b9142..f2ce073 100644
--- a/Train_simple_rgb2.py
+++ b/Train_simple_rgb2.py
@@ -99,7 +99,7 @@ time_now = datetime.now().strftime("%Y_%m_%d_%H_%M_%S")
 IS_WnB_enabled = True
 env_name = 'UR10eReachFixed-v13'
 
-training_steps = 2000000
+training_steps = 5000000
 ENTROPY = 0.01
 LR = linear_schedule(0.0003)
 CR = linear_schedule(0.1)
Submodule mj_envs contains modified content
diff --git a/mj_envs/robohive/envs/arms/reach_base_v2.py b/mj_envs/robohive/envs/arms/reach_base_v2.py
index c971904..01b2c0d 100644
--- a/mj_envs/robohive/envs/arms/reach_base_v2.py
+++ b/mj_envs/robohive/envs/arms/reach_base_v2.py
@@ -72,6 +72,7 @@ class ReachBaseV0(env_base_2.MujocoEnv):
         self.target_xyz_range = target_xyz_range
         self.IMAGE_WIDTH = image_width
         self.IMAGE_HEIGHT = image_height
+        self.rgb_out = np.ones((image_width, image_height))
         self.current_image = np.ones((image_width, image_height, 3), dtype=np.uint8)
         self.vel_action = [0]*6
         self.contact = 0 
@@ -92,6 +93,8 @@ class ReachBaseV0(env_base_2.MujocoEnv):
         obs_dict['reach_err'] = sim.data.site_xpos[self.target_sid]-sim.data.site_xpos[self.grasp_sid]
         obs_dict['goal_pos'] = sim.data.site_xpos[self.target_sid]
 
+        self.get_image_data()
+
         this_model = sim.model
         id_info = BodyIdInfo(this_model)
         this_data = sim.data
@@ -118,7 +121,7 @@ class ReachBaseV0(env_base_2.MujocoEnv):
         rgb, depth = copy.deepcopy(
             self.sim.renderer.render_offscreen(height=height,width=width,  camera_id=camera, depth = True)
         )
-
+        
         self.rgb_out = rgb
 
         rgb = cv.cvtColor(rgb, cv.COLOR_BGR2RGB)
@@ -201,6 +204,7 @@ class ReachBaseV0(env_base_2.MujocoEnv):
         # finalize step
         env_info = self.get_env_infos()
 
+
         obs = {'image': self.current_image.reshape((self.IMAGE_HEIGHT, self.IMAGE_WIDTH, 3)), 'vector': obs}
 
         return obs, env_info['rwd_'+self.rwd_mode], bool(env_info['done']), env_info
@@ -211,7 +215,7 @@ class ReachBaseV0(env_base_2.MujocoEnv):
         self.sim.model.site_pos[self.target_sid] = self.np_random.uniform(high=self.target_xyz_range['high'], low=self.target_xyz_range['low'])
         self.sim_obsd.model.site_pos[self.target_sid] = self.sim.model.site_pos[self.target_sid]
         obj_xyz_ranges = {
-            'object': {'low': [-0.05, -0.05, 0], 'high': [0.15, 0.15, 0]},
+            'object': {'low': [-0.85, -0.05, 0], 'high': [.85, 0.15, 0]},
         }
 
         new_x, new_y = np.random.uniform(
diff --git a/mj_envs/robohive/envs/arms/reach_base_v3.py b/mj_envs/robohive/envs/arms/reach_base_v3.py
index bc62107..5928bfd 100644
--- a/mj_envs/robohive/envs/arms/reach_base_v3.py
+++ b/mj_envs/robohive/envs/arms/reach_base_v3.py
@@ -72,6 +72,7 @@ class ReachBaseV0(env_base_2.MujocoEnv):
         self.target_xyz_range = target_xyz_range
         self.IMAGE_WIDTH = image_width
         self.IMAGE_HEIGHT = image_height
+        self.rgb_out = np.ones((image_width, image_height))
         self.current_image = np.ones((image_width, image_height, 3), dtype=np.uint8)
         self.vel_action = [0]*6
         self.contact = 0 
@@ -92,6 +93,8 @@ class ReachBaseV0(env_base_2.MujocoEnv):
         obs_dict['reach_err'] = sim.data.site_xpos[self.target_sid]-sim.data.site_xpos[self.grasp_sid]
         obs_dict['goal_pos'] = sim.data.site_xpos[self.target_sid]
 
+        self.get_image_data()
+
         this_model = sim.model
         id_info = BodyIdInfo(this_model)
         this_data = sim.data
@@ -118,7 +121,7 @@ class ReachBaseV0(env_base_2.MujocoEnv):
         rgb, depth = copy.deepcopy(
             self.sim.renderer.render_offscreen(height=height,width=width,  camera_id=camera, depth = True)
         )
-
+        
         self.rgb_out = rgb
 
         rgb = cv.cvtColor(rgb, cv.COLOR_BGR2RGB)
@@ -211,7 +214,7 @@ class ReachBaseV0(env_base_2.MujocoEnv):
         self.sim.model.site_pos[self.target_sid] = self.np_random.uniform(high=self.target_xyz_range['high'], low=self.target_xyz_range['low'])
         self.sim_obsd.model.site_pos[self.target_sid] = self.sim.model.site_pos[self.target_sid]
         obj_xyz_ranges = {
-            'object': {'low': [-0.05, -0.05, 0], 'high': [0.15, 0.15, 0]},
+            'object': {'low': [-0.85, -0.05, 0], 'high': [.85, 0.15, 0]},
         }
 
         new_x, new_y = np.random.uniform(
Submodule robohive/simhive/furniture_sim c098f76..cfb2387 (rewind):
diff --git a/videos/UR10ePickPlaceFixed-v0/2024_04_27_22_52_12front_video.mp4 b/videos/UR10ePickPlaceFixed-v0/2024_04_27_22_52_12front_video.mp4
deleted file mode 100644
index adac5ae..0000000
Binary files a/videos/UR10ePickPlaceFixed-v0/2024_04_27_22_52_12front_video.mp4 and /dev/null differ
diff --git a/videos/UR10ePickPlaceFixed-v0/2024_04_27_23_53_17front_video.mp4 b/videos/UR10ePickPlaceFixed-v0/2024_04_27_23_53_17front_video.mp4
deleted file mode 100644
index a3cdbbb..0000000
Binary files a/videos/UR10ePickPlaceFixed-v0/2024_04_27_23_53_17front_video.mp4 and /dev/null differ
diff --git a/videos/UR10ePickPlaceFixed-v0/2024_04_28_00_38_04front_video.mp4 b/videos/UR10ePickPlaceFixed-v0/2024_04_28_00_38_04front_video.mp4
deleted file mode 100644
index a01f1ee..0000000
Binary files a/videos/UR10ePickPlaceFixed-v0/2024_04_28_00_38_04front_video.mp4 and /dev/null differ
diff --git a/videos/UR10ePickPlaceFixed-v0/2024_04_28_17_04_14front_video.mp4 b/videos/UR10ePickPlaceFixed-v0/2024_04_28_17_04_14front_video.mp4
deleted file mode 100644
index 00f6449..0000000
Binary files a/videos/UR10ePickPlaceFixed-v0/2024_04_28_17_04_14front_video.mp4 and /dev/null differ
diff --git a/videos/UR10ePickPlaceFixed-v0/2024_04_29_10_31_11front_video.mp4 b/videos/UR10ePickPlaceFixed-v0/2024_04_29_10_31_11front_video.mp4
deleted file mode 100644
index 21b03cd..0000000
Binary files a/videos/UR10ePickPlaceFixed-v0/2024_04_29_10_31_11front_video.mp4 and /dev/null differ
diff --git a/videos/UR10ePickPlaceFixed-v0/2024_04_29_11_29_42front_video.mp4 b/videos/UR10ePickPlaceFixed-v0/2024_04_29_11_29_42front_video.mp4
deleted file mode 100644
index 997114c..0000000
Binary files a/videos/UR10ePickPlaceFixed-v0/2024_04_29_11_29_42front_video.mp4 and /dev/null differ
diff --git a/videos/UR10ePickPlaceFixed-v0/2024_05_02_14_26_27front_video.mp4 b/videos/UR10ePickPlaceFixed-v0/2024_05_02_14_26_27front_video.mp4
deleted file mode 100644
index 88bad3b..0000000
Binary files a/videos/UR10ePickPlaceFixed-v0/2024_05_02_14_26_27front_video.mp4 and /dev/null differ
diff --git a/videos/UR10ePickPlaceFixed-v0/2024_05_02_18_06_34front_video.mp4 b/videos/UR10ePickPlaceFixed-v0/2024_05_02_18_06_34front_video.mp4
deleted file mode 100644
index cfa3742..0000000
Binary files a/videos/UR10ePickPlaceFixed-v0/2024_05_02_18_06_34front_video.mp4 and /dev/null differ
diff --git a/videos/UR10ePickPlaceFixed-v0/2024_05_03_13_40_57front_video.mp4 b/videos/UR10ePickPlaceFixed-v0/2024_05_03_13_40_57front_video.mp4
deleted file mode 100644
index b76ae84..0000000
Binary files a/videos/UR10ePickPlaceFixed-v0/2024_05_03_13_40_57front_video.mp4 and /dev/null differ
diff --git a/videos/UR10ePickPlaceFixed-v0/2024_05_03_14_49_55front_video.mp4 b/videos/UR10ePickPlaceFixed-v0/2024_05_03_14_49_55front_video.mp4
deleted file mode 100644
index 5962742..0000000
Binary files a/videos/UR10ePickPlaceFixed-v0/2024_05_03_14_49_55front_video.mp4 and /dev/null differ
diff --git a/videos/UR10ePickPlaceFixed-v0/2024_05_03_15_23_56front_video.mp4 b/videos/UR10ePickPlaceFixed-v0/2024_05_03_15_23_56front_video.mp4
deleted file mode 100644
index 9ad9d27..0000000
Binary files a/videos/UR10ePickPlaceFixed-v0/2024_05_03_15_23_56front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v0/2024_04_27_19_03_05front_video.mp4 b/videos/UR10eReachFixed-v0/2024_04_27_19_03_05front_video.mp4
deleted file mode 100644
index 455a9af..0000000
Binary files a/videos/UR10eReachFixed-v0/2024_04_27_19_03_05front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v0/2024_04_30_18_32_34front_video.mp4 b/videos/UR10eReachFixed-v0/2024_04_30_18_32_34front_video.mp4
deleted file mode 100644
index acc8f70..0000000
Binary files a/videos/UR10eReachFixed-v0/2024_04_30_18_32_34front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v0/2024_05_01_09_33_44front_video.mp4 b/videos/UR10eReachFixed-v0/2024_05_01_09_33_44front_video.mp4
deleted file mode 100644
index daf01ad..0000000
Binary files a/videos/UR10eReachFixed-v0/2024_05_01_09_33_44front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v0/2024_05_01_10_55_51front_video.mp4 b/videos/UR10eReachFixed-v0/2024_05_01_10_55_51front_video.mp4
deleted file mode 100644
index 4f3a64c..0000000
Binary files a/videos/UR10eReachFixed-v0/2024_05_01_10_55_51front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v0/2024_05_01_11_11_08front_video.mp4 b/videos/UR10eReachFixed-v0/2024_05_01_11_11_08front_video.mp4
deleted file mode 100644
index 6308ec8..0000000
Binary files a/videos/UR10eReachFixed-v0/2024_05_01_11_11_08front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v0/2024_05_01_12_40_38front_video.mp4 b/videos/UR10eReachFixed-v0/2024_05_01_12_40_38front_video.mp4
deleted file mode 100644
index 01111f0..0000000
Binary files a/videos/UR10eReachFixed-v0/2024_05_01_12_40_38front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v0/2024_05_01_12_55_52front_video.mp4 b/videos/UR10eReachFixed-v0/2024_05_01_12_55_52front_video.mp4
deleted file mode 100644
index 3451072..0000000
Binary files a/videos/UR10eReachFixed-v0/2024_05_01_12_55_52front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v0/2024_05_01_13_33_14front_video.mp4 b/videos/UR10eReachFixed-v0/2024_05_01_13_33_14front_video.mp4
deleted file mode 100644
index 238b36a..0000000
Binary files a/videos/UR10eReachFixed-v0/2024_05_01_13_33_14front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v0/2024_05_01_13_59_50front_video.mp4 b/videos/UR10eReachFixed-v0/2024_05_01_13_59_50front_video.mp4
deleted file mode 100644
index d82b208..0000000
Binary files a/videos/UR10eReachFixed-v0/2024_05_01_13_59_50front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v0/2024_05_01_14_10_49front_video.mp4 b/videos/UR10eReachFixed-v0/2024_05_01_14_10_49front_video.mp4
deleted file mode 100644
index 9719ce3..0000000
Binary files a/videos/UR10eReachFixed-v0/2024_05_01_14_10_49front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v0/2024_05_01_14_20_38front_video.mp4 b/videos/UR10eReachFixed-v0/2024_05_01_14_20_38front_video.mp4
deleted file mode 100644
index aa5f458..0000000
Binary files a/videos/UR10eReachFixed-v0/2024_05_01_14_20_38front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v0/2024_05_01_14_35_51front_video.mp4 b/videos/UR10eReachFixed-v0/2024_05_01_14_35_51front_video.mp4
deleted file mode 100644
index 5ede918..0000000
Binary files a/videos/UR10eReachFixed-v0/2024_05_01_14_35_51front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v0/2024_05_01_18_59_50front_video.mp4 b/videos/UR10eReachFixed-v0/2024_05_01_18_59_50front_video.mp4
deleted file mode 100644
index 90f699f..0000000
Binary files a/videos/UR10eReachFixed-v0/2024_05_01_18_59_50front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v0/2024_05_01_19_18_48front_video.mp4 b/videos/UR10eReachFixed-v0/2024_05_01_19_18_48front_video.mp4
deleted file mode 100644
index 71443bf..0000000
Binary files a/videos/UR10eReachFixed-v0/2024_05_01_19_18_48front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v0/2024_05_01_20_51_26front_video.mp4 b/videos/UR10eReachFixed-v0/2024_05_01_20_51_26front_video.mp4
deleted file mode 100644
index 2b0d729..0000000
Binary files a/videos/UR10eReachFixed-v0/2024_05_01_20_51_26front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v0/2024_05_01_21_06_27front_video.mp4 b/videos/UR10eReachFixed-v0/2024_05_01_21_06_27front_video.mp4
deleted file mode 100644
index 3cedf1f..0000000
Binary files a/videos/UR10eReachFixed-v0/2024_05_01_21_06_27front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v0/2024_05_01_21_29_07front_video.mp4 b/videos/UR10eReachFixed-v0/2024_05_01_21_29_07front_video.mp4
deleted file mode 100644
index 11ef15c..0000000
Binary files a/videos/UR10eReachFixed-v0/2024_05_01_21_29_07front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v0/2024_05_01_22_15_54front_video.mp4 b/videos/UR10eReachFixed-v0/2024_05_01_22_15_54front_video.mp4
deleted file mode 100644
index fc97a6e..0000000
Binary files a/videos/UR10eReachFixed-v0/2024_05_01_22_15_54front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v0/2024_05_02_11_22_30front_video.mp4 b/videos/UR10eReachFixed-v0/2024_05_02_11_22_30front_video.mp4
deleted file mode 100644
index b88a01d..0000000
Binary files a/videos/UR10eReachFixed-v0/2024_05_02_11_22_30front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v0/2024_05_02_11_47_33front_video.mp4 b/videos/UR10eReachFixed-v0/2024_05_02_11_47_33front_video.mp4
deleted file mode 100644
index 34c01dd..0000000
Binary files a/videos/UR10eReachFixed-v0/2024_05_02_11_47_33front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v0/2024_05_02_12_45_37front_video.mp4 b/videos/UR10eReachFixed-v0/2024_05_02_12_45_37front_video.mp4
deleted file mode 100644
index 34a6b6a..0000000
Binary files a/videos/UR10eReachFixed-v0/2024_05_02_12_45_37front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v0/2024_05_02_13_19_02front_video.mp4 b/videos/UR10eReachFixed-v0/2024_05_02_13_19_02front_video.mp4
deleted file mode 100644
index 1d614a0..0000000
Binary files a/videos/UR10eReachFixed-v0/2024_05_02_13_19_02front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v0/2024_05_02_13_39_06front_video.mp4 b/videos/UR10eReachFixed-v0/2024_05_02_13_39_06front_video.mp4
deleted file mode 100644
index cd3bbdf..0000000
Binary files a/videos/UR10eReachFixed-v0/2024_05_02_13_39_06front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v0/2024_05_02_13_39_06video.mp4 b/videos/UR10eReachFixed-v0/2024_05_02_13_39_06video.mp4
deleted file mode 100644
index e7b7367..0000000
Binary files a/videos/UR10eReachFixed-v0/2024_05_02_13_39_06video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v0/2024_05_03_14_42_58front_video.mp4 b/videos/UR10eReachFixed-v0/2024_05_03_14_42_58front_video.mp4
deleted file mode 100644
index 481bd9a..0000000
Binary files a/videos/UR10eReachFixed-v0/2024_05_03_14_42_58front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v0/2024_05_03_15_26_32front_video.mp4 b/videos/UR10eReachFixed-v0/2024_05_03_15_26_32front_video.mp4
deleted file mode 100644
index f2b28ac..0000000
Binary files a/videos/UR10eReachFixed-v0/2024_05_03_15_26_32front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v1/2024_05_11_13_21_30front_video.mp4 b/videos/UR10eReachFixed-v1/2024_05_11_13_21_30front_video.mp4
deleted file mode 100644
index 79c896d..0000000
Binary files a/videos/UR10eReachFixed-v1/2024_05_11_13_21_30front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v1/2024_05_11_21_43_32front_video.mp4 b/videos/UR10eReachFixed-v1/2024_05_11_21_43_32front_video.mp4
deleted file mode 100644
index 81dc78c..0000000
Binary files a/videos/UR10eReachFixed-v1/2024_05_11_21_43_32front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v1/2024_05_11_21_52_53front_video.mp4 b/videos/UR10eReachFixed-v1/2024_05_11_21_52_53front_video.mp4
deleted file mode 100644
index 3be04bf..0000000
Binary files a/videos/UR10eReachFixed-v1/2024_05_11_21_52_53front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v1/2024_05_12_18_06_35front_video.mp4 b/videos/UR10eReachFixed-v1/2024_05_12_18_06_35front_video.mp4
deleted file mode 100644
index e0edba4..0000000
Binary files a/videos/UR10eReachFixed-v1/2024_05_12_18_06_35front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v1/2024_05_12_18_58_25front_video.mp4 b/videos/UR10eReachFixed-v1/2024_05_12_18_58_25front_video.mp4
deleted file mode 100644
index 969293f..0000000
Binary files a/videos/UR10eReachFixed-v1/2024_05_12_18_58_25front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v1/2024_05_12_19_18_06front_video.mp4 b/videos/UR10eReachFixed-v1/2024_05_12_19_18_06front_video.mp4
deleted file mode 100644
index d276b25..0000000
Binary files a/videos/UR10eReachFixed-v1/2024_05_12_19_18_06front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v1/2024_05_12_19_28_57front_video.mp4 b/videos/UR10eReachFixed-v1/2024_05_12_19_28_57front_video.mp4
deleted file mode 100644
index ed175d7..0000000
Binary files a/videos/UR10eReachFixed-v1/2024_05_12_19_28_57front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v1/2024_05_13_11_47_17front_video.mp4 b/videos/UR10eReachFixed-v1/2024_05_13_11_47_17front_video.mp4
deleted file mode 100644
index 459cd69..0000000
Binary files a/videos/UR10eReachFixed-v1/2024_05_13_11_47_17front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v1/2024_05_13_12_00_15front_video.mp4 b/videos/UR10eReachFixed-v1/2024_05_13_12_00_15front_video.mp4
deleted file mode 100644
index 7d94de3..0000000
Binary files a/videos/UR10eReachFixed-v1/2024_05_13_12_00_15front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v1/2024_05_13_12_12_37front_video.mp4 b/videos/UR10eReachFixed-v1/2024_05_13_12_12_37front_video.mp4
deleted file mode 100644
index 0f5d59e..0000000
Binary files a/videos/UR10eReachFixed-v1/2024_05_13_12_12_37front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v1/2024_05_13_23_28_12front_video.mp4 b/videos/UR10eReachFixed-v1/2024_05_13_23_28_12front_video.mp4
deleted file mode 100644
index 464a56a..0000000
Binary files a/videos/UR10eReachFixed-v1/2024_05_13_23_28_12front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v1/2024_05_14_15_34_00front_video.mp4 b/videos/UR10eReachFixed-v1/2024_05_14_15_34_00front_video.mp4
deleted file mode 100644
index 9bdd03c..0000000
Binary files a/videos/UR10eReachFixed-v1/2024_05_14_15_34_00front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v1/2024_05_14_16_57_04front_video.mp4 b/videos/UR10eReachFixed-v1/2024_05_14_16_57_04front_video.mp4
deleted file mode 100644
index 03c6c69..0000000
Binary files a/videos/UR10eReachFixed-v1/2024_05_14_16_57_04front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v1/2024_05_14_20_30_20front_video.mp4 b/videos/UR10eReachFixed-v1/2024_05_14_20_30_20front_video.mp4
deleted file mode 100644
index 73241f6..0000000
Binary files a/videos/UR10eReachFixed-v1/2024_05_14_20_30_20front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v1/2024_05_15_11_28_31front_video.mp4 b/videos/UR10eReachFixed-v1/2024_05_15_11_28_31front_video.mp4
deleted file mode 100644
index f0758fd..0000000
Binary files a/videos/UR10eReachFixed-v1/2024_05_15_11_28_31front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v1/2024_05_15_14_32_50+grab_1.mp4 b/videos/UR10eReachFixed-v1/2024_05_15_14_32_50+grab_1.mp4
deleted file mode 100644
index 9db0590..0000000
Binary files a/videos/UR10eReachFixed-v1/2024_05_15_14_32_50+grab_1.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v1/2024_05_15_14_32_50_grab_2.mp4 b/videos/UR10eReachFixed-v1/2024_05_15_14_32_50_grab_2.mp4
deleted file mode 100644
index 7005dad..0000000
Binary files a/videos/UR10eReachFixed-v1/2024_05_15_14_32_50_grab_2.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v1/2024_05_15_14_32_50front_video.mp4 b/videos/UR10eReachFixed-v1/2024_05_15_14_32_50front_video.mp4
deleted file mode 100644
index d8ad0ee..0000000
Binary files a/videos/UR10eReachFixed-v1/2024_05_15_14_32_50front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v1/2024_05_15_16_45_28end_video.avi b/videos/UR10eReachFixed-v1/2024_05_15_16_45_28end_video.avi
deleted file mode 100644
index e69de29..0000000
diff --git a/videos/UR10eReachFixed-v1/2024_05_15_16_45_28front_mask_video.mp4 b/videos/UR10eReachFixed-v1/2024_05_15_16_45_28front_mask_video.mp4
deleted file mode 100644
index 42b044d..0000000
Binary files a/videos/UR10eReachFixed-v1/2024_05_15_16_45_28front_mask_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v1/2024_05_15_16_45_28front_video.mp4 b/videos/UR10eReachFixed-v1/2024_05_15_16_45_28front_video.mp4
deleted file mode 100644
index dbd360c..0000000
Binary files a/videos/UR10eReachFixed-v1/2024_05_15_16_45_28front_video.mp4 and /dev/null differ
diff --git "a/videos/UR10eReachFixed-v1/\346\227\240\346\240\207\351\242\230\350\247\206\351\242\221\342\200\224\342\200\224\344\275\277\347\224\250Clipchamp\345\210\266\344\275\234 (2).mp4" "b/videos/UR10eReachFixed-v1/\346\227\240\346\240\207\351\242\230\350\247\206\351\242\221\342\200\224\342\200\224\344\275\277\347\224\250Clipchamp\345\210\266\344\275\234 (2).mp4"
deleted file mode 100644
index 368c5bf..0000000
Binary files "a/videos/UR10eReachFixed-v1/\346\227\240\346\240\207\351\242\230\350\247\206\351\242\221\342\200\224\342\200\224\344\275\277\347\224\250Clipchamp\345\210\266\344\275\234 (2).mp4" and /dev/null differ
diff --git a/videos/UR10eReachFixed-v2/2024_06_17_13_16_41front_mask_video.mp4 b/videos/UR10eReachFixed-v2/2024_06_17_13_16_41front_mask_video.mp4
deleted file mode 100644
index 2c2e3f5..0000000
Binary files a/videos/UR10eReachFixed-v2/2024_06_17_13_16_41front_mask_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v2/2024_06_17_13_16_41front_video.mp4 b/videos/UR10eReachFixed-v2/2024_06_17_13_16_41front_video.mp4
deleted file mode 100644
index 10138ff..0000000
Binary files a/videos/UR10eReachFixed-v2/2024_06_17_13_16_41front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v2/2024_06_17_18_45_32front_mask_video.mp4 b/videos/UR10eReachFixed-v2/2024_06_17_18_45_32front_mask_video.mp4
deleted file mode 100644
index f7b5284..0000000
Binary files a/videos/UR10eReachFixed-v2/2024_06_17_18_45_32front_mask_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v2/2024_06_17_18_45_32front_video.mp4 b/videos/UR10eReachFixed-v2/2024_06_17_18_45_32front_video.mp4
deleted file mode 100644
index df456ff..0000000
Binary files a/videos/UR10eReachFixed-v2/2024_06_17_18_45_32front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v2/2024_06_20_11_12_34front_mask_video.mp4 b/videos/UR10eReachFixed-v2/2024_06_20_11_12_34front_mask_video.mp4
deleted file mode 100644
index 12ce96c..0000000
Binary files a/videos/UR10eReachFixed-v2/2024_06_20_11_12_34front_mask_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v2/2024_06_20_11_12_34front_video.mp4 b/videos/UR10eReachFixed-v2/2024_06_20_11_12_34front_video.mp4
deleted file mode 100644
index 3471b39..0000000
Binary files a/videos/UR10eReachFixed-v2/2024_06_20_11_12_34front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v2/2024_06_21_08_41_26front_mask_video.mp4 b/videos/UR10eReachFixed-v2/2024_06_21_08_41_26front_mask_video.mp4
deleted file mode 100644
index 100057b..0000000
Binary files a/videos/UR10eReachFixed-v2/2024_06_21_08_41_26front_mask_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v2/2024_06_21_08_41_26front_video.mp4 b/videos/UR10eReachFixed-v2/2024_06_21_08_41_26front_video.mp4
deleted file mode 100644
index 704677a..0000000
Binary files a/videos/UR10eReachFixed-v2/2024_06_21_08_41_26front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v2/2024_06_21_23_59_31front_mask_video.mp4 b/videos/UR10eReachFixed-v2/2024_06_21_23_59_31front_mask_video.mp4
deleted file mode 100644
index b6329ce..0000000
Binary files a/videos/UR10eReachFixed-v2/2024_06_21_23_59_31front_mask_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v2/2024_06_21_23_59_31front_video.mp4 b/videos/UR10eReachFixed-v2/2024_06_21_23_59_31front_video.mp4
deleted file mode 100644
index 12a5d80..0000000
Binary files a/videos/UR10eReachFixed-v2/2024_06_21_23_59_31front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v2/2024_06_22_11_11_14front_mask_video.mp4 b/videos/UR10eReachFixed-v2/2024_06_22_11_11_14front_mask_video.mp4
deleted file mode 100644
index e7a0f99..0000000
Binary files a/videos/UR10eReachFixed-v2/2024_06_22_11_11_14front_mask_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v2/2024_06_22_11_11_14front_video.mp4 b/videos/UR10eReachFixed-v2/2024_06_22_11_11_14front_video.mp4
deleted file mode 100644
index 511c4e2..0000000
Binary files a/videos/UR10eReachFixed-v2/2024_06_22_11_11_14front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v2/2024_06_22_19_48_33front_mask_video.mp4 b/videos/UR10eReachFixed-v2/2024_06_22_19_48_33front_mask_video.mp4
deleted file mode 100644
index e3e192d..0000000
Binary files a/videos/UR10eReachFixed-v2/2024_06_22_19_48_33front_mask_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v2/2024_06_22_19_48_33front_video.mp4 b/videos/UR10eReachFixed-v2/2024_06_22_19_48_33front_video.mp4
deleted file mode 100644
index f89252a..0000000
Binary files a/videos/UR10eReachFixed-v2/2024_06_22_19_48_33front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v2/2024_06_23_02_03_41front_mask_video.mp4 b/videos/UR10eReachFixed-v2/2024_06_23_02_03_41front_mask_video.mp4
deleted file mode 100644
index fca4432..0000000
Binary files a/videos/UR10eReachFixed-v2/2024_06_23_02_03_41front_mask_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v2/2024_06_23_02_03_41front_video.mp4 b/videos/UR10eReachFixed-v2/2024_06_23_02_03_41front_video.mp4
deleted file mode 100644
index a721329..0000000
Binary files a/videos/UR10eReachFixed-v2/2024_06_23_02_03_41front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v2/2024_06_24_01_01_18front_mask_video.mp4 b/videos/UR10eReachFixed-v2/2024_06_24_01_01_18front_mask_video.mp4
deleted file mode 100644
index ff59faa..0000000
Binary files a/videos/UR10eReachFixed-v2/2024_06_24_01_01_18front_mask_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v2/2024_06_24_01_01_18front_video.mp4 b/videos/UR10eReachFixed-v2/2024_06_24_01_01_18front_video.mp4
deleted file mode 100644
index 6e9fc9d..0000000
Binary files a/videos/UR10eReachFixed-v2/2024_06_24_01_01_18front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v2/2024_06_24_11_10_04front_mask_video.mp4 b/videos/UR10eReachFixed-v2/2024_06_24_11_10_04front_mask_video.mp4
deleted file mode 100644
index eabba29..0000000
Binary files a/videos/UR10eReachFixed-v2/2024_06_24_11_10_04front_mask_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v2/2024_06_24_11_10_04front_video.mp4 b/videos/UR10eReachFixed-v2/2024_06_24_11_10_04front_video.mp4
deleted file mode 100644
index ecde798..0000000
Binary files a/videos/UR10eReachFixed-v2/2024_06_24_11_10_04front_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v2/2024_06_25_00_46_36front_mask_video.mp4 b/videos/UR10eReachFixed-v2/2024_06_25_00_46_36front_mask_video.mp4
deleted file mode 100644
index 714965b..0000000
Binary files a/videos/UR10eReachFixed-v2/2024_06_25_00_46_36front_mask_video.mp4 and /dev/null differ
diff --git a/videos/UR10eReachFixed-v2/2024_06_25_00_46_36front_video.mp4 b/videos/UR10eReachFixed-v2/2024_06_25_00_46_36front_video.mp4
deleted file mode 100644
index bc13ddf..0000000
Binary files a/videos/UR10eReachFixed-v2/2024_06_25_00_46_36front_video.mp4 and /dev/null differ
diff --git a/videos1/UR10eMask3C-v1/2024_09_13_15_26_302front_mask_video.mp4 b/videos1/UR10eMask3C-v1/2024_09_13_15_26_302front_mask_video.mp4
deleted file mode 100644
index 39327bb..0000000
Binary files a/videos1/UR10eMask3C-v1/2024_09_13_15_26_302front_mask_video.mp4 and /dev/null differ
diff --git a/videos1/UR10eMask3C-v1/2024_09_13_15_26_302front_video.mp4 b/videos1/UR10eMask3C-v1/2024_09_13_15_26_302front_video.mp4
deleted file mode 100644
index 8e661ea..0000000
Binary files a/videos1/UR10eMask3C-v1/2024_09_13_15_26_302front_video.mp4 and /dev/null differ
